Formula/gnu-cobol.rb

```diff
@@ -1,8 +1,8 @@
 class GnuCobol < Formula
-  desc "Implements much of the COBOL 85 and COBOL 2002 standards"
-  homepage "https://sourceforge.net/projects/open-cobol/"
-  url "https://downloads.sourceforge.net/project/open-cobol/gnucobol/2.2/gnucobol-2.2.tar.xz"
-  sha256 "dc18fc45c269debfe86a4bbe20a7250983cba6238ea1917e135df5926cd024a0"
+  desc "COBOL85-202x compiler supporting lots of dialect specific extensions"
+  homepage "https://www.gnu.org/software/gnucobol/"
+  url "https://alpha.gnu.org/gnu/gnucobol/gnucobol-3.1-rc1.tar.xz"
+  sha256 "c2e41c2ba520681a67c570d7246d25c31f7f55c8a145aaec3f6273a500a93a76"
   revision 1
```

Comment:
This should be removed

Reply:
What now - the revision? I guess "no revision" means zero, revision 1 means updated package, same source, rev2 the same, .. - correct?

Reply:
@GitMensch the revision only meaningful and useful when the version stays the same. In this case since we are doing a version bump, there is no need to have `revision` field.